api/HTMLIFrameElement.json

```diff
@@ -1113,10 +1113,10 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": "43"
+              "version_added": "20"
             },
             "chrome_android": {
-              "version_added": "18"
+              "version_added": "25"
```

Comment:
This is `srcdoc`, and this (and the mirrored data) is the only change in the PR that increases the `version_added`. This came from https://github.com/mdn/browser-compat-data/pull/1356 which just bumped the version number from "4", and "4" came from wiki migration in https://github.com/mdn/browser-compat-data/pull/2017, so I see no reason to especially trust it. `srcdoc` is not a feature that I can see being per-platform, it's deep inside HTML.